### PR TITLE
[feat] utility script to more conveniently download tutorial sample data

### PR DESCRIPTION
**Pull request by jono3030**
# Summary

- Adds utility script that let's user download sample datasets interactively
- Prompts to automatically extract and delete zip file(s)
- Cmd is available container wide from `/usr/bin`

### CODE REVIEW

**Review by anthonyprinaldi — COMMENTED** [review]
Looks great to me! Will be super useful. 

My only question is should we call this in our install script or mention it in the docs somewhere? Otherwise it will just go unnoticed 😢

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ RUN rm -rf $(python -c "from distutils.sysconfig import get_python_lib; print(ge
     pip install -e /code/ "botocore >= 1.20.110"
 ENV MIRACL_HOME=/code/miracl
 ENV ATLASES_HOME=/code/atlases
+COPY ./utility_scripts /usr/bin
```
[the remark this answers is not in the document]
Comment (anthonyprinaldi) [reply Dockerfile:15]: NVM this is correct

[the remark this answers is not in the document]
Comment (jono3030) [reply Dockerfile:15]: Good catch - I found that unclear as well when I implemented it. It copies the content of a directory into the directory that's designated as the target directory which I found intuitive. I built the image and ran the container to test it and it correctly copies the script into the `/bin/usr` folder. I needed to add `x+o` though to actually make it work as the script was owned by `root:root`.

[the remark this answers is not in the document]
Comment (jono3030) [reply Dockerfile:15]: Re: mentioning the script in the docs you are a 100% right - we'll definitely have to do that. I was thinking as part of the docs update PR?

**Review by anthonyprinaldi — APPROVED** [review]
Good to merge!

**Pull request merged**